microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -415,7 +415,7 @@ PWAmp is a music player PWA demo application that defines a widget. The PWAmp wi
 
 1. Follow the instructions in [enable widgets in Microsoft Edge](#enable-widgets-in-microsoft-edge).
 1. Go to [PWAmp](https://microsoftedge.github.io/Demos/pwamp/) and install the app on Windows 11.
-1. Open the Windows 11 widgets board by pressing **Ctrl+W**.
+1. Open the Windows 11 widgets board by pressing **🪟 Windows key+W**.
```

Comment:
```suggestion
1. Open the Windows 11 widgets board by pressing **Windows+W**.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts#key-names - says **Windows logo key**.  Doesn't provide good complete example to show viable spacing & +

full-text search of repo:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/4d15fbdf-13ec-451a-9096-d887b55a1706)
but latest convention per style guide is bold, no space around +

Reply:
fixed globally in repo